Builds fine (see above), but fails when I try to import the package:

```
~/.cache/nixpkgs-review/pr-205724-1/results/python310Packages.skorch/lib/python3.10/site-packages$ python -m skorch
Traceback (most recent call last):
  File "/nix/store/kph267nv0zvbgsgsvkz2a4ic5mgzc4cw-python3.9-numpy-1.23.3/lib/python3.9/site-packages/numpy/core/__init__.py", line 23, in <module>
    from . import multiarray
  File "/nix/store/kph267nv0zvbgsgsvkz2a4ic5mgzc4cw-python3.9-numpy-1.23.3/lib/python3.9/site-packages/numpy/core/multiarray.py", line 10, in <module>
    from . import overrides
  File "/nix/store/kph267nv0zvbgsgsvkz2a4ic5mgzc4cw-python3.9-numpy-1.23.3/lib/python3.9/site-packages/numpy/core/overrides.py", line 6, in <module>
    from numpy.core._multiarray_umath import (
ModuleNotFoundError: No module named 'numpy.core._multiarray_umath'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/nix/store/nzgqxw4d875pgl1lji8n3q82x3bf7ldi-python3-3.10.8/lib/python3.10/runpy.py", line 187, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/nix/store/nzgqxw4d875pgl1lji8n3q82x3bf7ldi-python3-3.10.8/lib/python3.10/runpy.py", line 146, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/nix/store/nzgqxw4d875pgl1lji8n3q82x3bf7ldi-python3-3.10.8/lib/python3.10/runpy.py", line 110, in _get_module_details
    __import__(pkg_name)
  File "/nix/store/932p8zd4sw2m0k2lb49z953wgps77aqn-python3.10-skorch-0.12.1/lib/python3.10/site-packages/skorch/__init__.py", line 9, in <module>
    from .history import History
  File "/nix/store/932p8zd4sw2m0k2lb49z953wgps77aqn-python3.10-skorch-0.12.1/lib/python3.10/site-packages/skorch/history.py", line 5, in <module>
    from skorch.utils import open_file_like
  File "/nix/store/932p8zd4sw2m0k2lb49z953wgps77aqn-python3.10-skorch-0.12.1/lib/python3.10/site-packages/skorch/utils.py", line 16, in <module>
    import numpy as np
  File "/nix/store/kph267nv0zvbgsgsvkz2a4ic5mgzc4cw-python3.9-numpy-1.23.3/lib/python3.9/site-packages/numpy/__init__.py", line 140, in <module>
    from . import core
  File "/nix/store/kph267nv0zvbgsgsvkz2a4ic5mgzc4cw-python3.9-numpy-1.23.3/lib/python3.9/site-packages/numpy/core/__init__.py", line 49, in <module>
    raise ImportError(msg)
ImportError: 

IMPORTANT: PLEASE READ THIS FOR ADVICE ON HOW TO SOLVE THIS ISSUE!

Importing the numpy C-extensions failed. This error can happen for
many reasons, often due to issues with your setup or how NumPy was
installed.

We have compiled some common reasons and troubleshooting tips at:

    https://numpy.org/devdocs/user/troubleshooting-importerror.html

Please note and check the following:

  * The Python version is: Python3.10 from "/nix/store/nzgqxw4d875pgl1lji8n3q82x3bf7ldi-python3-3.10.8/bin/python"
  * The NumPy version is: "1.23.3"

and make sure that they are the versions you expect.
Please carefully study the documentation linked above for further help.

Original error was: No module named 'numpy.core._multiarray_umath'
```

--- a/pkgs/development/python-modules/skorch/default.nix
+++ b/pkgs/development/python-modules/skorch/default.nix
@@ -2,8 +2,6 @@
 , buildPythonPackage
 , fetchPypi
 , pytestCheckHook
-, pytest
-, pytest-cov
 , flaky
 , numpy
 , pandas
@@ -16,24 +14,35 @@
 
 buildPythonPackage rec {
   pname = "skorch";
-  version = "0.11.0";
+  version = "0.12.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "b35cb4e50045742f0ffcfad33044af691d5d36b50212573753a804483a947ca9";
+    hash = "sha256-fjNbNY/Dr7lgVGPrHJTvPGuhyPR6IVS7ohBQMI+J1+k=";
   };
 
   propagatedBuildInputs = [ numpy torch scikit-learn scipy tabulate tqdm ];
-  checkInputs = [ pytest pytest-cov flaky pandas pytestCheckHook ];
+  checkInputs = [ flaky pandas pytestCheckHook ];
+
+  # patch out pytest-cov dep/invocation
+  postPatch = ''
+    substituteInPlace setup.cfg  \
+      --replace "--cov=skorch" ""  \
+      --replace "--cov-report=term-missing" ""  \
+      --replace "--cov-config .coveragerc" ""
+  '';
 
   disabledTests = [
     # on CPU, these expect artifacts from previous GPU run
     "test_load_cuda_params_to_cpu"
     # failing tests
     "test_pickle_load"
-    "test_grid_search_with_slds_"
-    "test_grid_search_with_dict_works"
   ];
+
+  # tries to import `transformers` and download HuggingFace data
+  disabledTestPaths = [ "skorch/tests/test_hf.py" ];
+
+  pythonImportsCheck = [ "skorch" ];
 
   meta = with lib; {
     description = "Scikit-learn compatible neural net library using Pytorch";
@@ -41,7 +50,5 @@ buildPythonPackage rec {
     changelog = "https://github.com/skorch-dev/skorch/blob/master/CHANGES.md";
     license = licenses.bsd3;
     maintainers = with maintainers; [ bcdarwin ];
-    # TypeError: __init__() got an unexpected keyword argument 'iid'
-    broken = true;
   };
 }

